gradle.properties

```diff
@@ -10,7 +10,7 @@ gradle.publish.key=xx
 gradle.publish.secret=xx
 
 # See https://dev.to/jmfayard/configuring-gradle-with-gradle-properties-211k
-org.gradle.parallel=false
+org.gradle.parallel=true
```

Comment:
https://scans.gradle.com/s/pywo4r5glfll2/tests/failed

add a override to kotest-extensions:kotest-extensions-http